Library/Homebrew/utils/github.rb

```diff
@@ -234,6 +234,7 @@ def open_api(url, data: nil, request_method: nil, scopes: [].freeze)
         json
       end
     rescue JSON::ParserError => e
+      return output if not_json
```

Comment:
```suggestion
      return output unless parse_json
```

and I'd suggest it be after `return if http_code == "204"` to avoid unnecessary parsing.